mehdibo/oauth2-fortytwo/1.0/manifest.json

```diff
@@ -0,0 +1,6 @@
+{
+    "env": {
+        "OAUTH_FT_ID": "APPLICATION_ID",
+        "OAUTH_FT_SECRET": "APPLICATION_SECRET"
```

Comment:
I dont see where these are used.

Reply:
OKey. I see. 

I think we should add a recipe for the knpy/oauth2-client instead then

Reply:
Ok I will try to work on one once I got sometime, I will close this PR